packages/react-core/src/demos/PrimaryDetail.md

```diff
@@ -1369,14 +1369,14 @@ class PrimaryDetailCardView extends React.Component {
                 aria-label={anySelected ? 'Deselect all' : 'Select all'}
                 isChecked={areAllSelected}
                 onClick={this.splitCheckboxSelectAll}
-              ></DropdownToggleCheckbox>
+              >
+                {numSelected !== 0 && (
+                  <React.Fragment>{allSelected ? this.state.totalItemCount : numSelected} selected</React.Fragment>
```

Comment:
Nit: this could be updated to match the other two demos:

```suggestion
                  <React.Fragment>{numSelected} selected</React.Fragment>
```

Since `numSelected` when all are selected should be the same as `totalItemCount`